astropy/modeling/functional_models.py

```diff
@@ -91,6 +91,32 @@ class Gaussian1D(Fittable1DModel):
     amplitude = Parameter(default=1)
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
+    _bbox = 'auto'
+
+    @property
+    def bbox(self):
```

Comment:
After checking again how other model properties are implemented, we could think about a more explicit solution to declaring the bounding box. A concrete suggestion would be to return a dictionary that contains a 2-tuple for every input parameter name, like so `{'x': (-5, 5)}`.